package.json

```diff
@@ -36,7 +36,6 @@
     "@types/chai": "^4.2.11",
     "@types/mocha": "^7.0.2",
     "@types/node": "^10.0.3",
-    "@types/yargs-parser": "^15.0.0",
```

Comment:
what's the motivation for vendoring `@types/yargs-parser`, no objection just curious?

Reply:
Some types were incomplete, and the factory function was missing.

As @QmarkC is working on `yargs-parser` migration to typescript too, I didn't find it useful to fix upstream typings in `@types/yargs-parser` which will soon be dropped.